### ISSUE TITLE
Emails are using numbers instead of names

### ISSUE THREAD

**Issue by Drew-Casey** [ecsion/eSeaGrant-Client-Bug-Tracker#1045]
eSeaGrant is again using a number instead of names for email. 

Example: 
"Happy Monday 5501, 
This is just a friendly reminder that you have a report that is past due on eSeaGrant..."

**Comment by Drew-Casey** [comment]
FYI - Now emails not including any coded information.

**Comment by Dipali-Ecsion** [comment]
Can you please provide me email template name and link

**Comment by Drew-Casey** [comment]
Link 1: http://eseagrant.texasseagrant.org/index.php?__vtrftk=sid%3Aee686c9f2c550f2f8f2664f14af706f0bbafe03d%2C1589300566&selected_ids=%5B%225909%22%2C%225910%22%2C%225914%22%2C%226281%22%2C%22100403%22%2C%22100768%22%2C%22122972%22%5D&excluded_ids=&viewname=&module=Emails&view=ComposeEmail&callreportingid=125408&rfpphaseid=&search_key=&operator=&search_value=&search_params=null&fieldModule=Projects&to=%5B%5D&selectedFields=%5B%22project_email%22%2C%22delegate_pi_email%22%5D&triggerEventName=postSelection8835

Email Template 1: Extension Report is Past Due

Link 2: http://eseagrant.texasseagrant.org/index.php?__vtrftk=sid%3Ab320f48ba8530422f2556fbd54267f0f433e8c9c%2C1589300659&selected_ids=%5B%225476%22%5D&excluded_ids=&viewname=&module=Emails&view=ComposeEmail&callreportingid=125176&rfpphaseid=&search_key=&operator=&search_value=&search_params=null&fieldModule=Projects&to=%5B%5D&selectedFields=%5B%22project_email%22%2C%22delegate_pi_email%22%5D&triggerEventName=postSelection5331

Email Template 2: RQP Report Available

The second email template had difficulty recognizing email address in the "To" field.

**Comment by Drew-Casey** [comment]
The problem persists. I used this email template (https://eseagrant.texasseagrant.org/index.php?module=EmailTemplates&view=Detail&record=85&type=Normal) and it outputted numbers again. One email did not have a name at all. 

Here is the email it sent:
"Hello 5499, 
You have a monthly metrics report that is past due on eSeaGrant. 
Regards..."

**Comment by Drew-Casey** [comment]
I just tested the generic email function found on the "Select Projects" page and it worked as expected. I attempted to test the "Due Dates (Beta)" email function by clicking on the send reminders now button for the same report call. It only pulled a few of the emails associated with the account but it did use the PI name. (FYI - PI names are reversed on the Due Dates email). 

![image](https://user-images.githubusercontent.com/44243756/88827691-e45c3c80-d18f-11ea-884b-96310456dd53.png)
![image](https://user-images.githubusercontent.com/44243756/88827733-f211c200-d18f-11ea-9e6d-7b22d7005092.png)
